firefox_ui_tests/remote/security/test_untrusted_connection_error_page.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette_driver import By, Wait
+from marionette_driver.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestUntrustedConnectionErrorPage(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://ssl-selfsigned.mozqa.com'
+
+    def tearDown(self):
+        FirefoxTestCase.tearDown(self)
+
+    def test_untrusted_connection_error_page(self):
+        # In some localized builds, the default page redirects
+        target_url = self.browser.get_final_url(self.browser.default_homepage)
+
+        # Test the GetMeOutOfHereButton from an Untrusted Error page
+        with self.marionette.using_context('content'):
+            self.assertRaises(MarionetteException, self.marionette.navigate, self.url)
+            # Wait for the DOM to receive events
+            time.sleep(1)
+
+            button = self.marionette.find_element(By.ID, "getMeOutOfHereButton")
+            button.click()
+            Wait(lambda mn: target_url == mn.get_url())
```

Comment:
The problem here is how you call this. `Wait()` is a constructor and requires a reference to marionette. On this class there is a `until()` method, which wants a callback for the condition. With this fixed it should work.